calcom/cal.com
# fix: don't show team unaccepted members

## What does this PR do?

When adding team members, unaccepted members will now not show up as options. This is just a feature for organizations. Also fixes the broken page when adding members in a subteam from the team creation wizard, which had an unstyled border.

## Type of change


## How should this be tested?

Try to create a regular team, add members and create a Collective/Round robin event type, see that pending members will not be an option. On the other hand, for org subteams, pending members will be an option when creating a subteam event-type.

## Mandatory Tasks
